src/javascript/PageComposer/PageComposer.jsx

```diff
@@ -134,10 +137,17 @@ export default function () {
         };
     }, []);
 
-    const iFrameOnLoad = () => {
+    const [errorPage, setErrorPageorPage] = useState(false);
+
+    const iFrameOnLoad = e => {
         if (window.frames['page-composer-frame'] !== undefined) {
             window.addEventListener('message', iFrameOnHistoryMessage, false);
             updateStoreAndHistory(getPathFromChildIFrame());
+
+            let element = e.target.contentWindow.document.querySelector('head meta[name=\'description\']');
+            if (element && element.attributes.content.value.startsWith('40')) {
+                setErrorPageorPage(true);
```

Comment:
Is there a reason why it is called setErrorPageorPage instead of just setErrorPage ?

Reply:
no :-)